use std::error::Error;


fn main() -> Result<(), Box<dyn Error>>
{
    println!("Hello Plot!");

    Ok(())
}